src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -573,7 +573,7 @@ bool BaseCouplingScheme::measureConvergence(
   bool oneSuffices  = false;
   PRECICE_ASSERT(_convergenceMeasures.size() > 0);
   if (not utils::MasterSlave::isSlave()) {
-    _convergenceWriter->writeData("Timestep", _timesteps);
+    _convergenceWriter->writeData("TimeWindow", _timesteps);
```

Comment:
There is still the issue of `_timesteps`. Going down that rabbit hole?

Reply:
That part, we postponed. All internal variables are still named "timestep".

Reply:
https://github.com/precice/precice/issues/639